src/components/segmentedControl/useSegmentedControlPreset.ts

```diff
@@ -0,0 +1,54 @@
+import {type SegmentedControlProps} from './';
+import {BorderRadiuses, Colors} from '../../style/';
+import {SegmentProps} from './segment';
+import type {ColorValue} from 'react-native';
+
+interface useSegmentedControlPresetProps extends SegmentedControlProps, Partial<SegmentProps> {
```

Comment:
Why are you adding "props" at the end? Is "useSegmentedControlPreset' not enough?

Reply:
you mean `Partial<SegmentProps>`?

Reply:
No, I mean the name of the hook